pkg/controller/deadmanssnitch/deadmanssnitch_controller_test.go

```diff
@@ -460,6 +474,26 @@ func verifyNoSyncSet(c client.Client, expected *SyncSetEntry) bool {
 	return true
 }
 
+func verifyNoSecret(c client.Client, expected *SecretEntry) bool {
```

Comment:
Take a look again at testing `Secret` deletion.  I commented out this block that deleted the secret and all tests still passed.  I would have expected it to fail.  Not sure why at a glance.

https://github.com/openshift/deadmanssnitch-operator/pull/44/files#diff-cbcc4d0350863bcdb11e774f6cacbaf5R339-R345